makefile

```diff
@@ -219,6 +219,18 @@ doc-example-names = $(example-names:%=doc/examples/%.html)
 
 doc-lib-names = $(lib-names:%=doc/lib/%.html)
 
+tutorial-data = t10k-images-idx3-ubyte t10k-labels-idx1-ubyte
+tutorial-data := $(tutorial-data:%=examples/%)
+
+$(tutorial-data):
+	wget -qO $@.gz http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/$(@F).gz
```

Comment:
Since we're fetching these data over http, would you mind adding a `sha256sum` integrity check?  I know it's kind of annoying, but using an unsigned file off the internet without at least checking that it's the same as it was when the makefile was written gives me the willies.

Reply:
@axch - done!